src/services/video-conference-manager/index.ts

```diff
@@ -92,15 +94,23 @@ export default class VideoConfereceManager {
       disableCameraOverlay,
     } = options;
 
-    let { position, camerasOrientation, skipMeetingSettings, devices } = options;
+    let { layoutPosition, position, camerasPosition, skipMeetingSettings, devices } = options;
 
     if (browserService.isMobileDevice) {
-      position = 'bottom';
-      camerasOrientation = 'horizontal';
+      position = CamerasPosition.BOTTOM;
+      camerasPosition = CamerasPosition.BOTTOM;
     }
 
-    if (camerasOrientation === 'horizontal') {
-      position = 'bottom';
+    if (camerasPosition === CamerasPosition.TOP || camerasPosition === CamerasPosition.BOTTOM) {
+      position = camerasPosition;
+    } else if (!browserService.isMobileDevice) {
+      position = camerasPosition;
+      if ((layoutPosition === LayoutPosition.LEFT) && (camerasPosition === CamerasPosition.RIGHT)) {
+        layoutPosition = LayoutPosition.RIGHT;
+      }
+      if ((layoutPosition === LayoutPosition.RIGHT) && (camerasPosition === CamerasPosition.LEFT)) {
+        layoutPosition = LayoutPosition.LEFT;
+      }
```

Comment:
The complexity of these conditions has increased a lot, I think it is time to make a separate function that returns the correct orientation of the cameras. 

By putting in a different function you will also be able to remove these conditionals from within other conditionals